src/index.ts

```diff
@@ -0,0 +1,65 @@
+import es from 'event-stream';
+import type { AggregationCursor, Document, FindCursor } from 'mongodb';
+import type { Readable } from 'stream';
+
+import stream from './stream';
+import type { SchemaParseOptions, Schema } from './stream';
+import * as schemaStats from './stats';
+
+type MongoDBCursor = AggregationCursor | FindCursor;
+
+/**
+ * Convenience shortcut for parsing schemas. Accepts an array, stream or
+ * MongoDB cursor object to parse documents` from.
+ */
+function parseSchema(
+  docs: Document[] | MongoDBCursor | Readable,
+  options?: SchemaParseOptions
+): Promise<Schema> {
+  const promise = new Promise<Schema>((resolve, reject) => {
```

Comment:
minor, but you can probably just `return new Promise(...)` here. Can't see the promise being used in this function.